packages/acceptance-tests/pkg-tests-fixtures/packages/nm-express/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "nm-express",
+  "version": "1.0.0",
+  "dependencies": {
+    "nm-core": "1.0.0"
+  },
+  "peerDependencies": {
+    "graphql": "1 || 2"
+  }
+}
```

Comment:
Wouldn't `peer-deps-lvl1` work? That would avoid having to create new fixtures:

https://github.com/yarnpkg/berry/blob/master/packages/acceptance-tests/pkg-tests-fixtures/packages/peer-deps-lvl1-1.0.0/package.json#L1-L10

Reply:
Done